bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4469,10 +4469,11 @@
             UPDATE concept SET is_set = FALSE WHERE concept_id = @concept_id;
         </sql>
     </changeSet>
-    <changeSet id="BAM-20180830150635" author="Angshu" runOnChange="true" context="release0.91">
-        <comment>Override SQL query for Get Patient Details in Ward</comment>
-        <sqlFile path="BahmniBMPatientListInWard.sql"/>
-    </changeSet>
+    <!-- would be handled as part of BAH-1935 -->
```

Comment:
any particular reason this changeset is commented? How is this linked with the bean creation issue?

Reply:
Hi @angshu,
global property query commented because it was causing SQL syntax issue,
track this issue with https://bahmni.atlassian.net/browse/BAH-1935